Examples/MAX78002/CSI2/utils.h

```diff
@@ -30,8 +30,8 @@
  * ownership rights.
  *
  ******************************************************************************/
-#ifndef EXAMPLES_MAX78002_CSI2_UTILS_H_
-#define EXAMPLES_MAX78002_CSI2_UTILS_H_
+#ifndef _UTILS_H_
+#define _UTILS_H_
```

Comment:
Surprised the linter didn't complain.

Reply:
A lot of files didn't change, so the linter skipped them.  Just fixed in all linter errors